### PR TITLE
refactor(base64): Factor base64 out of SwingSet network

### PR DESCRIPTION
**Pull request by kriskowal**
This change moves code from SwingSet kernel into a package of its own such that it can be shared with `import-bundle`.

Commits are individually reviewable and will be squashed to pass the conventional commits check after approval to land.

### CODE REVIEW

File: packages/SwingSet/src/vats/network/network.js [hunk 1]
```diff
@@ -85,7 +85,7 @@ export const makeConnection = (
         .then(
           ack => {
             pendingAcks.delete(ackDeferred);
-            ackDeferred.resolve(toBytes(ack));
+            ackDeferred.resolve(toBytes(String(ack)));
```
[the remark this answers is not in the document]
Comment (michaelfig) [reply packages/SwingSet/src/vats/network/network.js]: `ack` is a string or undefined.  The cast is intentional.

[the remark this answers is not in the document]
Comment (kriskowal) [reply packages/SwingSet/src/vats/network/network.js]: Ah, so `toBytes` should lift `undefined` to an empty slice of bytes?

[the remark this answers is not in the document]
Comment (kriskowal) [reply packages/SwingSet/src/vats/network/network.js]: Ah, LSP believes `ack` is `string | true | Buffer | ArrayBuffer`. The `true` is leaking from the short-circuit in the above catch. Pushing a fix.

**Review by michaelfig — APPROVED** [review]
LGTM!  Thanks for using the types.

**Pull request merged**